src/test/scala/mesosphere/marathon/core/task/bus/TaskStatusUpdateTestHelper.scala

```diff
@@ -135,10 +136,11 @@ object TaskStatusUpdateTestHelper {
     }
   }
 
-  def killed(instance: Instance = defaultInstance) = {
+  def killed(instance: Instance = defaultInstance, draining: Boolean = false) = {
```

Comment:
so apart from tests is there a piece of code that actually sets draining=true?

Reply:
This is a test fixture method, so no – only test code uses test fixtures :D